### PR TITLE
[EDU-5093] update bucket journey

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

File: src/content/docs/en/pages/store-journey/storage/update-bucket.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+description: Learn how to update the buckets in your account using the Azion API and Azion CLI.
+permalink: /documentation/products/store/storage/update-buckets/
+meta_tags: Edge Storage, Storage, Cloud, S3, Bucket, File Storage, Object Storage, API, Azion CLI, Update Buckets
+namespace: docs_store_journey_storage_update_buckets
+menu_namespace: storeMenu
+---
+
+import Tabs from '~/components/tabs/Tabs'
+
+In this guide, we will walk you through the process of updating an Edge Storage bucket using both the Azion API and Azion CLI. This includes changing the permissions of a bucket, which can be crucial for managing access to your data.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/update-bucket.mdx:12]:
```suggestion
This guide walks you through updating an Edge Storage bucket using the Azion API and Azion CLI. It includes changing the permissions of a bucket, which can be crucial for managing access to your data. 
```

File: src/content/docs/en/pages/store-journey/storage/update-bucket.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+meta_tags: Edge Storage, Storage, Cloud, S3, Bucket, File Storage, Object Storage, API, Azion CLI, Update Buckets
+namespace: docs_store_journey_storage_update_buckets
+menu_namespace: storeMenu
+---
+
+import Tabs from '~/components/tabs/Tabs'
+
+In this guide, we will walk you through the process of updating an Edge Storage bucket using both the Azion API and Azion CLI. This includes changing the permissions of a bucket, which can be crucial for managing access to your data. 
+
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/update-bucket.mdx:14]:
```suggestion
```

File: src/content/docs/en/pages/store-journey/storage/update-bucket.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+azion update edge-storage bucket
+```
+
+Azion will present a series of interactions, so the bucket can be updated. Run `azion update edge-storage bucket -h` for further information about the data that can be updated. 
+
+</Fragment>
+
+<Fragment slot="panel.api">
+
+In this example, the permission of the bucket is changed.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/update-bucket.mdx:40]:
```suggestion
In this example, the permission of the bucket will be changed. 
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/store-journey/storage/update-bucket.mdx:40]: This section explains how to do it, it isn't already changed

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/store-journey/storage/update-bucket.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    <Fragment slot="tab.api">API</Fragment>
+
+<Fragment slot="panel.cli">
+
+## Requirements 
+
+- [Azion CLI installed](/en/documentation/products/azion-cli/overview/#installing-azion-cli). 
+- [A personal token configured](/en/documentation/devtools/cli/globals/#token).
+
+To create a bucket:
```
Comment (GabrielAzion) [inline src/content/docs/en/pages/store-journey/storage/update-bucket.mdx:28]: update

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/store-journey/storage/update-bucket.mdx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/store-journey/storage/update-bucket.mdx:27]:
```suggestion
To update a bucket: 
```

**Pull request merged**